AppCenter/AppCenter/Internals/Context/AuthToken/MSAuthTokenContextDelegate.h

```diff
@@ -11,8 +11,9 @@ NS_ASSUME_NONNULL_BEGIN
  *
  * @param authTokenContext The auth token context.
  * @param authToken The auth token.
+ * @param isNewUser True if the user has changed, false otherwise.
```

Comment:
Per my previous comment (https://github.com/Microsoft/AppCenter-SDK-Apple/pull/1328/files#r260102518), "is New User" != "has User changed".